SampleSize/SampleSizeCalculation.md

```diff
@@ -0,0 +1,379 @@
+library(presize)
+prec_prop(p = 0.36, conf.width = 0.09)
+```
+
+    ## 
+    ##      sample size for a proportion with Wilson confidence interval. 
+    ## 
+    ##      p      padj        n conf.width conf.level       lwr       upr
+    ## 1 0.36 0.3612296 433.5577       0.09       0.95 0.3162296 0.4062296
+    ## 
+    ## NOTE: padj is the adjusted proportion, from which the ci is calculated.
+
+If we know that we can only afford to follow 300 participants, we can
+see what the confidence interval would be in that case too, via the `n`
+argument (instead of `cond.width`):
+
+``` r
+prec_prop(p = 0.36, n = 300)
+```
+
+    ## 
+    ##      precision for a proportion with Wilson confidence interval. 
+    ## 
+    ##      p    padj   n conf.width conf.level       lwr       upr
+    ## 1 0.36 0.36177 300  0.1080014       0.95 0.3077693 0.4157707
+    ## 
+    ## NOTE: padj is the adjusted proportion, from which the ci is calculated.
+
+It’s also possible to check multiple scenarios, by passing multiple
+values to a parameter. Below we vary the number of participants (what CI
+width is possible to determine with that number of participants?):
+
+``` r
+prec_prop(p = .36, n = c(300, 350, 400, 450, 450))
+```
+
+    ## 
+    ##      precision for a proportion with Wilson confidence interval. 
+    ## 
+    ##      p      padj   n conf.width conf.level       lwr       upr
+    ## 1 0.36 0.3617700 300 0.10800136       0.95 0.3077693 0.4157707
+    ## 2 0.36 0.3615199 350 0.10007265       0.95 0.3114836 0.4115562
+    ## 3 0.36 0.3613317 400 0.09366763       0.95 0.3144979 0.4081655
+    ## 4 0.36 0.3611850 450 0.08835346       0.95 0.3170083 0.4053617
+    ## 5 0.36 0.3611850 450 0.08835346       0.95 0.3170083 0.4053617
+    ## 
+    ## NOTE: padj is the adjusted proportion, from which the ci is calculated.
+
+The results can also be plotted easily, which is particularly useful
+when running multiple scenarios.
+
+``` r
+library(ggplot2)  # for plotting
+library(magrittr) # for 'piping'
+
+prec_prop(p = .36, n = c(300, 350, 400, 450, 500)) %>% 
+  as.data.frame() %>% 
+  ggplot(aes(x = n, y = conf.width)) +
+  geom_line() +
+  labs(x = "N participants", 
+       y = "CI width")
+```
+
+![](SampleSizeCalculation_files/figure-gfm/unnamed-chunk-8-1.png)<!-- -->
+
+For additional details and examples, see the [`presize`
+website](https://ctu-bern.github.io/presize/).
+
+# Conclusion
+
+Sample size calculation is a very important part of the study planning
+process. To make the most of your meeting with the statistician, have
+the answers to the questions mentioned above.
+
+# Quiz answers
+
+Question 1: A (it’s a false positive).  
+Question 2: B or C (a larger sample increases power which reduces the
+risk of a type 2 error).  
+Question 3: C (you’re not testing a difference, just trying the estimate
+the quantity to within a given limit).  
+Question 4: B, C, and D (these are all important. E can also be useful
+for approximating trial duration.
```

Comment:
I found several typos (e.g. here the parenthesis is not closed): is there a way that I can correct them directly to save time?